src/QuestionDisplay/QuestionDisplay.tsx

```diff
@@ -1,16 +1,26 @@
-import { getQuizQuestions } from "../getQuizQuestions/getQuizQuestions";
+import { getQuizQuestions, Answer, Question } from "../getQuizQuestions/getQuizQuestions";
+import { Quiz } from "../Quiz/Quiz";
 
-export function QuestionDisplay() {
-    const questions = getQuizQuestions();
+interface QuestionDisplayProp{
+    question: Question;
+}
 
-    const question = questions[Math.floor(Math.random() * questions.length)];
+export function QuestionDisplay(props: QuestionDisplayProp) {
+    // const questions = getQuizQuestions();
```

Comment:
leaving old code in as comments creates a risk of confusion when it comes to re-reading/updating the code.
please remove these